MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>use markdown titles to structure the notebook</p><p> </p><p>e.g.</p><p> </p><pre class="ql-syntax" spellcheck="false"># Title of the notebook

## Load data

## Exploration

### "Urbanité Ruralité"
</pre><p> </p><p>can even sometime generate TOC</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>